modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/modals/ImportStructureModal.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayButton from '@clayui/button';
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayModal, {useModal} from '@clayui/modal';
+import PropTypes from 'prop-types';
+import React, {useRef} from 'react';
+
+const ImportStructureModal = ({namespace, onClose: onCloseProp}) => {
+	const {observer, onClose} = useModal({
+		onClose: onCloseProp,
+	});
+	const inputFileRef = useRef();
+
+	return (
+		<ClayModal observer={observer} size="md">
+			<ClayModal.Header>
+				{Liferay.Language.get('import-structure')}
+			</ClayModal.Header>
+			<ClayModal.Body>
+				<ClayAlert displayType="info" title="Info:">
+					{Liferay.Language.get(
+						'once-you-click-import-the-process-will-run-in-the-background-this-may-take-a-while'
+					)}
+				</ClayAlert>
+				<ClayForm.Group>
+					<label htmlFor="nameInputText">
+						{Liferay.Language.get('name')}
+					</label>
+					<ClayInput
+						id="nameInputText"
+						name={`${namespace}_name`}
+						type="text"
+					/>
+					<label htmlFor="jsonInputText">
+						{Liferay.Language.get('json-file')}
+					</label>
+					<ClayInput.Group>
+						<ClayInput.GroupItem prepend>
+							<ClayInput id="jsonInputText" type="text" />
+						</ClayInput.GroupItem>
+						<ClayInput.GroupItem append shrink>
+							<ClayButton
+								displayType="secondary"
+								onClick={() => inputFileRef.current.click()}
+							>
+								{Liferay.Language.get('select')}
+							</ClayButton>
+						</ClayInput.GroupItem>
+						<ClayInput.GroupItem shrink>
+							<ClayButton displayType="secondary">
+								{Liferay.Language.get('clear')}
+							</ClayButton>
+						</ClayInput.GroupItem>
+					</ClayInput.Group>
+				</ClayForm.Group>
+				<ClayInput
```

Comment:
ClayInput is an styled component and you don't  need to use that, since you intent to hide it (no style will be necessary in this case). You can simplify the code and improve its performance using the regular HTML `input` tag:
```suggestion
				<input
```